Refs #26029 -- Deprecated DEFAULT_FILE_STORAGE and STATICFILES_STORAGE settings. (#15610)
django/django: docs/howto/static-files/deployment.txt

The basic outline of putting static files into production consists of two
steps: run the :djadmin:`collectstatic` command when static files change, then
arrange for the collected static files directory (:setting:`STATIC_ROOT`) to be

> moved to the static file server and served. Depending on ``staticfiles`` from
> :setting:`STORAGES`, files may need to be moved to a new location

Reviewer: ```suggestion
moved to the static file server and served. Depending the ``staticfiles``
:setting:`STORAGES` alias, files may need to be moved to a new location
```

(Again here, but you see the idea.)
